scripts/release.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+if [ -z "$OLD_VERSION" ]; then
+    echo "A release requires OLD_VERSION to be defined"
+    exit 1
+fi
+
+if [ "$OLD_VERSION" != "none" ]; then
+    comparison="$OLD_VERSION..HEAD"
+fi
+pre_release="-p"
+
+if [ "$OLD_VERSION" != "none" ];  then
+    changelog=$(git log $comparison --oneline --no-merges --reverse)
+
+    if [ -z "$changelog" ]; then
+        echo "No new changes to release!"
+        exit 0
+    fi
+else
+    changelog="don't forget to update the changelog"
+fi
+
+set -x
+( ( github-release -v release $pre_release -r auth_proxy -t $BUILD_VERSION -d "**Changelog**<br/>$changelog" ) ) || exit 1
```

Comment:
I don't see any issue with doing something like:

```sh
[ -n "`which github-release`" ] || go get -u github.com/aktau/github-release
```

at the top of the script.  If we don't do that, we need to document that that is the specific version we require (though we should probably do that anyways)